src/styles/utility.scss

```diff
@@ -0,0 +1,24 @@
+.auth-view-container {
```

Comment:
Nie rozjechały się tutaj wcięcia? IMO też powinny być na dwa taby. To raczej standard.

Reply:
Yup. W VSC nie miałem ustawionego prettiera dla scss.